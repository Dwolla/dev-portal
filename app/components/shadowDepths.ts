// Box shadow depths
import {
  BALANCE_DARK,
  CONNECT_DARK,
  PURPLE_012,
  PURPLE_014,
  PURPLE_020,
} from "./colors";

export const BOX_SHADOW_1 = "0 1px 2px 0 rgba(31,54,78,0.12)";
export const BOX_SHADOW_2 = "0 2px 4px 0 rgba(31,54,78,0.12)";
export const BOX_SHADOW_3 = "0 4px 8px 0 rgba(31,54,78,0.12)";
export const BOX_SHADOW_4 = "0 8px 14px 0 rgba(31,54,78,0.12)";
export const BOX_SHADOW_5 = "0 15px 25px 0 rgba(31,54,78,0.12)";
export const BOX_SHADOW_6 = "0 3px 5px 0 rgba(53, 65, 83, 0.24)";
export const BOX_SHADOW_7 = "0 15px 25px 0 rgba(218, 225, 232, 0.7)";

export const BOX_SHADOW_8 = `0px 5px 5px -3px ${PURPLE_020}, 0px 8px 10px 1px ${PURPLE_014}, 0px 3px 14px 2px ${PURPLE_012};`;

export const BALANCE_ICON_SHADOW = `0px 0px 15px ${BALANCE_DARK}`;
export const CONNECT_ICON_SHADOW = `0px 0px 15px ${CONNECT_DARK}`;
