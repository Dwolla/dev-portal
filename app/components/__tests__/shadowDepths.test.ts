import * as shadowDepths from "../shadowDepths";

test("shadowDepths", () => {
  expect(shadowDepths).toMatchSnapshot();
});
