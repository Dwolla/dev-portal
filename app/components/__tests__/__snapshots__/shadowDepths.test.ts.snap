// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shadowDepths 1`] = `
{
  "BALANCE_ICON_SHADOW": "0px 0px 15px #00475E",
  "BOX_SHADOW_1": "0 1px 2px 0 rgba(31,54,78,0.12)",
  "BOX_SHADOW_2": "0 2px 4px 0 rgba(31,54,78,0.12)",
  "BOX_SHADOW_3": "0 4px 8px 0 rgba(31,54,78,0.12)",
  "BOX_SHADOW_4": "0 8px 14px 0 rgba(31,54,78,0.12)",
  "BOX_SHADOW_5": "0 15px 25px 0 rgba(31,54,78,0.12)",
  "BOX_SHADOW_6": "0 3px 5px 0 rgba(53, 65, 83, 0.24)",
  "BOX_SHADOW_7": "0 15px 25px 0 rgba(218, 225, 232, 0.7)",
  "BOX_SHADOW_8": "0px 5px 5px -3px rgba(31, 31, 50, 0.2), 0px 8px 10px 1px rgba(31, 31, 50, 0.14), 0px 3px 14px 2px rgba(31, 31, 50, 0.12);",
  "CONNECT_ICON_SHADOW": "0px 0px 15px #631F3E",
}
`;
